e2e/tests/components/select/custom-heading/select-custom-heading.po.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { browser, by, element, ElementFinder } from 'protractor';
+
+export class SelectCustomHeadingPage {
```

Comment:
Can you remove the unused code in this class? Additionally, all functions should have a return type.